app/templates/README.mdown

```diff
@@ -69,3 +69,67 @@ Or on Windows:
 The application comes with a sample set of JSON data. The detail controller,
 the part that handles showing your data, can not only handle JSON, but also
 XML, Binary, and Text data out of the box.
+
+## Running with PM2
+
+To run you must install pm2 with:
+
+`npm install pm2 -g`
+
+Make sure you have the file `process.json` if it's missing then run:
```

Comment:
Since `process.json` is not a default name for pm2, I vote calling it `boot.json`, since that is a pattern that should be relatively familiar to slush users.